### DISCUSSION TITLE
Remove new tab page shortcuts

### DISCUSSION THREAD

**Discussion by Bri-an** [bromite/bromite#1738] [General]
I've looked all over and don't see any way to remove the recent pages shortcuts that automatically get added to the new tab page. If there's not a way, can I suggest here that it be added?

**Comment by uazo** [comment]
I don't think it's available.
As a workaround you can try to change the homepage by entering `about:blank`

**Reply by Bri-an** [reply]
Thanks. That works. I always had home page disabled, so I had to enable it, set to about:blank, and also enable the "use for new tabs" option.
